crates/kitsune_p2p/kitsune_p2p/src/test.rs

```diff
@@ -41,6 +41,7 @@ mod tests {
     }
 
     #[tokio::test(flavor = "multi_thread")]
+    // @neonphog Is this test of any use?
```

Comment:
not from here, we should delete this as written, we're testing gets elsewhere, right?

Reply:
Yes in holochain_p2p